src/Conference/AddConferenceForm.test.js

```diff
@@ -1,51 +1,61 @@
+jest.mock("../api", () => ({
```

Comment:
Out of interest why did you move this here? I thought they were hoisted by jest anyway so there wasn't a need to move them above imports

Reply:
I think Karam was told by someone that it was good practice to move the jest mocks to before the imports. We're happy to move them back though if they should be below?

Reply:
No thats fine, I don't use jest so they probably know more than me :)